### ISSUE TITLE
Problem with nested tag/label

### ISSUE THREAD

**Issue by reyman** [nylas/N1#2918]
##### What operating system are you using?
Linux

##### What version of N1 are you using?

0.4.56-816edde

**Bug?**

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
Gmail

My nested tag in gmail doesn't appear in Nylas, as you can see in the two screenshoot attached.

![aaa](https://cloud.githubusercontent.com/assets/466329/19265135/2b025714-8fa4-11e6-9a77-a51caf6af63c.jpg)
![aa](https://cloud.githubusercontent.com/assets/466329/19265134/2b021a42-8fa4-11e6-9b71-a26b0d991e83.jpg)

**Comment by bengotow** [comment]
Hey! Thanks for reporting this - is there a label with just `2-Projects` as it's name? It looks like N1 didn't see or didn't sync the parent label, so it's not nesting the children. We should probably change the nesting algorithm so it doesn't require the parent to be present and just splits on the `/`...

**Comment by reyman** [comment]
Yes, i have a label `2-Projets` only used to regroup nested labels `P-Archeo`, etc..
For example, gmail code label P-H24 like that : `2-Projets/R-EIGHTIES/P-H24` and for gmail search like that `label:2-projets-r-eighties-p-h24 and `
